python_modules/dagster/dagster_tests/core_tests/hook_tests/test_hook_def.py

```diff
@@ -465,3 +466,17 @@ def run_success_hook():
 
     assert called.get("hook_one") == 2
     assert called.get("hook_two") == 1
+
+
+def test_hook_context_op_solid_provided():
+    @success_hook()
+    def hook_one(context):
+        pass
+
+    @op
+    def hook_op(_):
+        pass
+
+    with pytest.raises(check.CheckError):
+        ctx = build_hook_context(op=hook_op, solid=hook_op)
+        hook_one(ctx)
```

Comment:
Nit: I think this is dead code, I think you just need to call `build_hook_context`

Reply:
Thanks, corrected this!